source/integration_tools.html.md

```diff
@@ -0,0 +1,73 @@
+# Integration Tools
+
+Integration Tools is a section of Workshop (Welkin's codeless configuration tool), which allows you to configure Apps and other integrations. You can configure which systems have access to data within Welkin and how they access it.
+
+# Overview
+
+Integration Tools allows developers and program directors to manage their integrations with Welkin without contacting a member of the Welkin professional services team. This is a self serve tool for managing how data flows out of Welkin via Welkin's APIs.
+
+Integration Tools is broken into 3 main sections. Credentials for setting up access credentials. Webhooks for getting notifcations about changes to data in Welkin. Environments for selecting which Welkin instance/environment you want to configure integrations for.
+
+# Credentials
```

Comment:
Do we want to bring up the interaction between credentials and TPAs at all (i.e. the fact that invalidating a credential might cause problems for a TPA, as we recently discovered)?